packages/common/mui/src/lib/feature-loader/feature-loader.tsx

```diff
@@ -0,0 +1,39 @@
+import { ReactElement } from 'react';
+import { Box } from '@mui/material';
+import { PageLoading } from '../page-loading/page-loading';
+import { useNotional } from '@notional-finance/notionable-hooks';
+
+interface FeatureLoaderProps {
+  children: ReactElement<any, any>;
+  featureLoaded?: boolean;
+  backgroundColor?: string;
+}
+
+export const FeatureLoader = ({
+  children,
+  featureLoaded = true,
+  backgroundColor,
```

Comment:
Should this just inherit from the dark / light setting from the current app theme? What happens if it is unset?

Reply:
This prop is just for the landing page which we force to be dark. Within the rest of the app it is unset and therefor defaults to the light or dark theme setting